dwave/preprocessing/include/dwave/presolve.h

```diff
@@ -253,7 +277,11 @@ void Presolver<bias_type, index_type, assignment_type>::apply() {
         if (!changes) break;
         changes = false;
 
-        // *-- todo: clear out 0 variables/interactions in the constraints
+        // *-- clear out 0 variables/interactions in the constraints and objective
+        remove_zero_biases(model_.objective);
```

Comment:
should we remove the zero biases before the main loop here? Unless it is possible to modify the bias of variables during at different iterations..

Reply:
I think right now there aren't any other techniques that will result in 0 biases, but there will be in the future so may as well put it in the correct place.

Reply:
so, in case we remove a variable with no bias, should the `remove_zero_bias` function set `changes` to `false`?

Reply:
Added